src/test/tests/utils/testIntegration.test.ts

```diff
@@ -0,0 +1,105 @@
+import { assert } from 'chai';
+
+import { ConnectionManager } from '../../../main/connect/connectionManager';
+import { LogManager } from '../../../main/log/logManager';
+import { ScanUtils } from '../../../main/utils/scanUtils';
+import { createTestConnectionManager } from './utils.test';
+import { Resource } from '../../../main/utils/resource';
+import { BinaryRunner } from '../../../main/scanLogic/scanRunners/binaryRunner';
+
+export class BaseIntegrationEnv {
+    public static readonly ENV_PLATFORM_URL: string = 'JFROG_IDE_PLATFORM_URL';
+    public static readonly ENV_ACCESS_TOKEN: string = 'JFROG_IDE_ACCESS_TOKEN';
+
+    public readonly logManager: LogManager = new LogManager().activate();
+
+    private _connectionManager!: ConnectionManager;
+
+    public static directory: string;
+
+    public async initialize() {
+        // Initialize connection manager
+        this._connectionManager = await createTestConnectionManager(this.logManager);
+        // Don't override existing connection details
+        let cacheUrl: string | undefined = process.env[ConnectionManager.URL_ENV];
+        let cacheToken: string | undefined = process.env[ConnectionManager.ACCESS_TOKEN_ENV];
```

Comment:
Cache is something else
```suggestion
        let oldUrl: string | undefined = process.env[ConnectionManager.URL_ENV];
        let oldToken: string | undefined = process.env[ConnectionManager.ACCESS_TOKEN_ENV];
```